app/queries/search_results_query.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+CONDITION_URL_EQUALS = '? = ANY (adwords_top_urls) OR ? = ANY (non_adwords_urls)'
```

Comment:
We can use the [`Placeholder Conditions`](https://guides.rubyonrails.org/active_record_querying.html#placeholder-conditions) to make it a bit clearer

```suggestion
CONDITION_URL_EQUALS = ':url = ANY (adwords_top_urls) OR :url = ANY (non_adwords_urls)'
```

And you can provide the name explicitly:

```ruby
@search_results.where(CONDITION_URL_EQUALS, url: url)
```